daemon/snap.go

```diff
@@ -55,27 +55,27 @@ func snapDate(info *snap.Info) time.Time {
 }
 
 // localSnapInfo returns the information about the current snap for the given name plus the SnapState with the active flag and other snap revisions.
-func localSnapInfo(st *state.State, name string) (info *snap.Info, active bool, err error) {
+func localSnapInfo(st *state.State, name string) (info *snap.Info, snapstOut *snapstate.SnapState, err error) {
```

Comment:
Actually, in this case the return names call all go away, because the types are extremely clear about what the results contain. I'll pick it up and fix it to get it in.